Class-12-(Model-Validation-and-Enums).md

```diff
@@ -22,23 +23,25 @@
 
    b. How can bad data be dangerous?
 
+   c. Why would using a ViewModel be beneficial in model validation?
+
 1. Be sure that the students can differentiate between client and server side validation tools.
-1. Discuss some commonly annotations from the [csharp Validation API](https://csharpee.github.io/csharpee-spec/csharpdocs/csharpx/validation/constraints/package-summary.html).
-1. Walk the students through the *validation flow* mentioned in the text, perhaps with a new sample field on the ``Event`` model from ``coding-events`` .
+1. Discuss some commonly attributes from the [C#](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations?view=netcore-3.1).
```

Comment:
this maybe a more appropriate link to guide this discussion: https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-3.1#built-in-attributes

Reply:
Done!